### PR TITLE
feat:Added home page

### PR DESCRIPTION
**Pull request by Hardiegogo**
Added home page:
image
image
image

### CODE REVIEW

File: css_files/home_page.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  font-size: 2rem;
+  display: block;
+  width: fit-content;
+}
+
+.hero-left-content{
+  margin-top: 10rem;
+  width: fit-content;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (fahd23) [inline css_files/home_page.css:86]: One suggestion, you can use shorthand for this like, margin: 10rem auto 0; am I right?

**Review by fahd23 — APPROVED** [review]
The code looks clean and UI is awesome🔥but i cann't find home page in preview

File: css_files/home_page.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+.btn-badge{
+    background-color: white;
+    color: var(--primary-color);
+    z-index: 9;
+  }
+  
+.badge-wrapper-i,.icons{
+    cursor: pointer;
+    font-size: 2.4rem;
```
Comment (stutisk) [inline css_files/home_page.css:58]: You can use variables for font size

**Review by stutisk — APPROVED** [review]

**Comment by Hardiegogo** [comment]
> The code looks clean and UI is awesome🔥but i cann't find home page in preview

Yes I will add the link to the homepage.

File: css_files/home_page.css (hunk 2 above)
Comment (Hardiegogo) [reply css_files/home_page.css:58]: Okay

**Pull request merged**